### ISSUE TITLE
botting failed

### ISSUE THREAD

**Issue by cheekyboiii** [KasperiP/kiderat-desktop#17]
i was trying to buy tickets to a big event but the bot kept trying but it never succeeded. I saw kide had done something to the original event link so i suppose they had some extra protection agains bots this year. can you try to figure out what went wrong and maybe do a new patch before 10.11? would be appriciated

**Comment by KasperiP** [comment]
What was the event you tried? I just tested the bot and it seems to be working.

**Comment by cheekyboiii** [comment]
Glöggrundan 2023, yes the bot works for "ordinary" events but im pretty sure kide had done something on their part for this one. For example the event link was way longer than usual and you had to get the shorter version of the link by copying it from the sharing page.

**Comment by KasperiP** [comment]
Yeah, that event seems to have a different data format compared to the "usual" events. It includes linkedProductVariants, which I don't think I've seen before. This event appears to be related to group selling, so it's possible that this specific detail is intended for such cases. It might also be a bug that has always been present in the bot, but we just haven't noticed it because it's quite unusual to encounter sales like this. 

Will investigate this further ASAP.
